CHANGELOG.md

```diff
@@ -4,13 +4,41 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Common Changelog](https://common-changelog.org/).
 
+[1.2.0]: https://github.com/sablier-labs/v2-core/compare/v1.1.2...v1.2.0
 [1.1.2]: https://github.com/sablier-labs/v2-core/compare/v1.1.1...v1.1.2
 [1.1.1]: https://github.com/sablier-labs/v2-core/compare/v1.1.0...v1.1.1
 [1.1.0]: https://github.com/sablier-labs/v2-core/compare/v1.0.2...v1.1.0
 [1.0.2]: https://github.com/sablier-labs/v2-core/compare/v1.0.1...v1.0.2
 [1.0.1]: https://github.com/sablier-labs/v2-core/compare/v1.0.0...v1.0.1
 [1.0.0]: https://github.com/sablier-labs/v2-core/releases/tag/v1.0.0
 
+## [1.2.0] - 2024-06-16
+
+### Changed
+
+- **Breaking:** Move common logic into `Lockup` contract ([#813](https://github.com/sablier-labs/v2-core/pull/813),
+  [#850](https://github.com/sablier-labs/v2-core/pull/850), [#784](https://github.com/sablier-labs/v2-core/pull/784))
```

Comment:
shouldn't we also include the fix [PR](https://github.com/sablier-labs/v2-core/pull/941)) ?

Reply:
I included that PR in the point "Disallow zero `startTime`" because I thought its more relevant there.

Reply:
why is it relevant there? that PR introduces the check to `blockTimestamp` not zero

imo, it is relevant here, as the main PR 813, introduces a bug, which is fixed in 941

Reply:
You are right.

Reply:
Resolved.